Sources/UInt128.swift

```diff
@@ -339,30 +300,21 @@ extension UInt128: UnsignedInteger {
         self.init(value.toUIntMax())
     }
     public func toUIntMax() -> UIntMax {
-        return UIntMax(value.lowerBits)
+        return lo.toUIntMax()
     }
     // MARK: Hashable Conformance
     public var hashValue: Int {
-        return Int(
-            value.lowerBits.hashValue ^ value.upperBits.hashValue
-        )
-    }
-    // MARK: ForwardIndexType Conformance
```

Comment:
While successor() and predecessor() usage is deprecated in the Swift 3.1 stdlib, I only want to remove them during a major version bump. After your other changes our ironed out and merged, I wouldn't mind looking at having these removed in a subsequent PR.